sdk/analytics/src/main/java/com/klaviyo/analytics/Klaviyo.kt

```diff
@@ -361,28 +382,67 @@ object Klaviyo {
     /**
      * Checks whether a notification intent originated from Klaviyo
      */
+    @JvmStatic
     @Deprecated(
         "Use isKlaviyoNotificationIntent instead, will be removed in the next major version",
         ReplaceWith("isKlaviyoNotificationIntent")
     )
-    val Intent.isKlaviyoIntent: Boolean get() = this.isKlaviyoNotificationIntent
+    fun isKlaviyoIntent(intent: Intent): Boolean = intent.isKlaviyoNotificationIntent
+
+    /**
+     * Checks whether a notification intent originated from Klaviyo
+     */
+    @Deprecated(
+        "Use isKlaviyoNotificationIntent instead, will be removed in the next major version",
+        ReplaceWith("isKlaviyoNotificationIntent")
+    )
+    val Intent.isKlaviyoIntent: Boolean
+        @JvmName("_isKlaviyoIntent")
```

Comment:
ick, don't like these hackarounds, just stick to the method java intends which is to create methods in place of the extension property